sphinx/themes/basic/static/searchtools.js_t

```diff
@@ -256,7 +256,7 @@ var Search = {
             displayNextItem();
           });
         } else if (DOCUMENTATION_OPTIONS.HAS_SOURCE) {
-          $.ajax({url: DOCUMENTATION_OPTIONS.URL_ROOT + '_sources/' + item[0] + '.txt',
+          $.ajax({url: DOCUMENTATION_OPTIONS.URL_ROOT + '_sources/' + item[5] + '.txt',
```

Comment:
This line always appends `.txt`. This should be replaced with `html_source_suffix`.

In addition, this script fails searching if `source_suffix` is `.txt`. searchtools.js expects `foo.txt.txt`, but sphinx generates `foo.txt` because of `source_suffix` equals to `html_source_suffix`. I think special treatment for `.txt` is not needed. Certainly `.txt.txt` is ugly, but it is simple to know the path to the original file.